Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/approximate_convex_decomposition.h

```diff
@@ -1394,9 +1364,82 @@ void merge(std::vector<Convex_hull<GeomTraits>>& candidates, std::vector<int8_t>
 }
 
 }
-
+/**
+ * \ingroup PMP_convex_decomposition_grp
+ *
+ * \brief approximates the input mesh by a number of convex hulls. The input mesh is voxelized and the convex hull of the mesh is hierarchically split until a volume threshold is met.
+ *        Afterwards, a greedy pair-wise merging combines smaller convex hulls until the given number of convex hulls is met.
+ *
+ * \tparam FaceGraph a model of `HalfedgeListGraph`, `FaceListGraph`, and `MutableFaceGraph`
+ *
+ * \tparam OutputIterator must be an output iterator accepting variables of type `std::pair<std::vector<geom_traits::Point_3>, std::vector<std::array<unsigned int, 3> > >`.
+ *
+ * \tparam NamedParameters a sequence of \ref bgl_namedparameters "Named Parameters"
+ *
+ * \param mesh the input mesh to approximate by convex hulls
+ * \param hulls_out output iterator into which convex hulls are recorded
+ * \param np an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
+ *
+ * \cgalNamedParamsBegin
+ *
+ *   \cgalParamNBegin{maximum_number_of_voxels}
+ *     \cgalParamDescription{Gives an upper bound of the number of voxels. The largest bounding box side will have a length of the cubic root of `maximum_number_of_voxels`.}
+ *     \cgalParamType{unsigned int}
+ *     \cgalParamDefault{1000000}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{maximum_depth}
+ *     \cgalParamDescription{maximum depth of hierarchical splits}
+ *     \cgalParamType{unsigned int}
+ *     \cgalParamDefault{10}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{maximum_number_of_convex_hulls}
+ *     \cgalParamDescription{maximum number of convex hulls produced by the method}
+ *     \cgalParamType{unsigned int}
+ *     \cgalParamDefault{16}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{volume_error}
+ *     \cgalParamDescription{maximum difference in percent of volume of the local convex hull with the sum of voxel volumes. If surpassed, the convex hull will be split.}
+ *     \cgalParamType{double}
+ *     \cgalParamDefault{0.01}
+ *   \cgalParamNEnd
+ *
+ *   \cgalParamNBegin{split_at_concavity}
+ *     \cgalParamDescription{split the local box of the convex hull in the mid of the longest axis (faster) or search the concavity along the largest axis for splitting.}
```

Comment:
Is longest and largest the same?  Then use only one word.

Reply:
fixed